xyz-hub-service/src/main/java/com/here/xyz/httpconnector/config/JDBCExporter.java

```diff
@@ -77,7 +77,7 @@ public static Future<Export.ExportStatistic> executeExport(Export j, String sche
 
                                     return executeParallelExportAndCollectStatistics( j, promise, exportFutures);
                                 }catch (SQLException e){
-                                    logger.warn(e);
+                                    logger.warn("job[{}] {}", j.getId(), e.getMessage());
```

Comment:
That way the stack-trace will be missing in the logs. Please replace `e.getMessage()` by `e` and also remove the second `{}` in the message text. Otherwise the effect would be the same.